common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/BadgeConstants.kt

```diff
@@ -0,0 +1,37 @@
+package com.trafi.ui.component.internal
+
+import com.trafi.ui.theme.internal.BasicFontWeight
+import com.trafi.ui.theme.internal.ColorPalette.Gray300
+import com.trafi.ui.theme.internal.CurrentTheme
+import com.trafi.ui.theme.internal.copy
+import com.trafi.ui.theme.internal.dp
+
+class BadgeConstants(theme: CurrentTheme) {
+    val defaultContentColor = theme.colorPalette.onPrimary
+    val disabledColor = Gray300
+
+    val textStyleMedium = theme.typographyScale.textM.copy(fontWeight = BasicFontWeight.Bold)
+    val textStyleSmall = theme.typographyScale.textS.copy(fontWeight = BasicFontWeight.Bold)
+
+    val minHeightMedium = 24.dp
+    val minHeightSmall = 16.dp
+
+    val cornerRadiusMedium = theme.cornerRadiusScale.mediumBadgeRadius
+    val cornerRadiusSmall = theme.cornerRadiusScale.smallBadgeRadius
```

Comment:
If you agree that we don't want to let SDK users theme these globally, just inline the values here, i.e. write `cornerRadiusMedium = CornerRadiusScale.xs`